### ISSUE TITLE
redundant query parameters

### ISSUE THREAD

**Issue by matthewberryman** [urbanriskmap/petabencana-docs#1]
Not sure why https://github.com/urbanriskmap/petabencana-docs/blob/master/routes/floods.md has both format and geoformat, since specifying a geoformat also specifies which format (GeoJSON and TopoJSON are JSON format and CAP is XML format).

**Comment by mgvarley** [comment]
Yup was thinking the same when I was writing them.  Tom - one to discuss next week


>

**Comment by matthewberryman** [comment]
@mgvarley You beat me to it before I had a chance to assign @talltom (now done).

**Comment by tomasholderness** [comment]
@matthewberryman Have discussed with @mgvarley - he's working on the endpoint currently.
